engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlanConfiguration.java

```diff
@@ -1229,7 +2135,7 @@ private String getSimpleName(Attribute<?> attribute) {
   }
 
   private Map<String,String> getApplicationProperties(List<AppConf> appConfs){
-    Map<String, String> appProps = new HashMap<String, String>();
+    Map<String, String> appProps = Maps.newHashMap();
```

Comment:
Why are we using this factory method here instead of instantiating a HashMap directly?